libpod/boltdb_state_internal.go

```diff
@@ -535,7 +535,9 @@ func (s *BoltState) addContainer(ctr *Container, pod *Pod) error {
 
 	// JSON container structs to insert into DB
 	// TODO use a higher-performance struct encoding than JSON
-	configJSON, err := json.Marshal(ctr.config)
+	configJSON := []byte{}
```

Comment:
Why the change here?

Reply:
leftover from old iteration, sorry will remove.